### ISSUE TITLE
Bug: $localize is not a function - error with dialog component

### ISSUE THREAD

**Issue by daniel-sc** [sbb-design-systems/sbb-angular#1565]
**Describe the bug**
localization errors:
![image](https://user-images.githubusercontent.com/117919/177189558-ea6357a6-6a62-4b34-a49c-d57c365312b7.png)

as discussed @kyubisation

**Comment by daniel-sc** [comment]
The following workaround
```json
"postinstall": "find node_modules/@sbb-esta -type f -name \"*.mjs\" -exec sed -r -i 's/^(const .* = )\\$localize.*;/\\1\\\"\\\";/g' {} +"
```
does _not_ fully resolve this issue - then I get the following error:
![image](https://user-images.githubusercontent.com/117919/177275912-da503ba9-fc1c-40b1-8c67-fe18b748be7c.png)

**Comment by kyubisation** [comment]
Thanks for the update. I'm currently analyzing this issue.

**Comment by daniel-sc** [comment]
for future reference: the following workaround seems to work fine (of course all component texts stay in english!)
```json
"postinstall": "find node_modules/@sbb-esta -type f -name \"*.mjs\" -exec sed -r -i 's/\\$localize\\s?`[^`]*:([^`:]*)`/\\\"\\1\\\"/g' {} +"
```

**Comment by daniel-sc** [comment]
We are not alone: https://github.com/angular/angular/issues/46503 (not conclusive, though..)

**Comment by Alderman8** [comment]
After I updated to Angular 14, I found a similar issue in my project, preventing me from accessing some pages. I've noticed that this error pops out when I try to access a page with ngb-timepicker element so it must be related to that in my case. Localization of the rest of my project works fine. Any suggestion on how to handle this? 

![localize_issue](https://user-images.githubusercontent.com/61386463/177585362-84d95f34-dd2a-4df9-ad65-964b86e23e09.PNG)
